plugins/data-management/src/HierarchicalTrackSelectorWidget/components/tree/HamburgerMenu.tsx

```diff
@@ -93,22 +82,34 @@ export default observer(function HamburgerMenu({
 
   return (
     <>
-      <IconButton
-        className={classes.menuIcon}
-        onClick={event => setMenuEl(event.currentTarget)}
-      >
-        <MenuIcon />
-      </IconButton>
-
-      <JBrowseMenu
-        anchorEl={menuEl}
-        open={Boolean(menuEl)}
-        onMenuItemClick={(_, callback) => {
-          callback()
-          setMenuEl(undefined)
-        }}
-        onClose={() => setMenuEl(undefined)}
+      <CascadingMenuButton
         menuItems={[
+          {
+            label: 'Sort tracks by name',
+            type: 'checkbox',
+            checked: model.hSortTrackNames,
+            onClick: () => model.setSortTrackNames(!model.hSortTrackNames),
+          },
+          {
+            label: 'Sort categories by name',
+            type: 'checkbox',
+            checked: model.hSortCategories,
+            onClick: () => model.setSortCategories(!model.hSortCategories),
+          },
+          { type: 'divider' },
+          {
+            label: 'Collapse subcategories',
```

Comment:
I think this should be a toggle to expand/collapse all subcategories

Reply:
by toggle do you mean checkbox? i avoided checkbox because you can get into states that are not 'all or nothing', e.g. you could expand all, then collapse one manually, so then it's not in an expand all state

Reply:
it is somewhat tempting to make it a toggle, but i kind of prefer the current behavior because it makes it clear that it's an acton that manipulates the state, and that you can repeat it if you want, rather than being a true toggle. i tried to in the below comment remove the subcategories option when no subcategories are available, to reduce cognitive load